content/blog/how-to-create-a-ci-cd-pipeline-to-execute-automation-test-suite-with-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How to Set Up an Automation Pipeline to Achieve Observability on HPE
```

Comment:
Original title sounds better to me. And publishing is easier without modifying it.